src/Apps/Purchase/Components/PurchaseHistory.tsx

```diff
@@ -0,0 +1,144 @@
+import {
+  Box,
+  Button,
+  Flex,
+  Image,
+  Link,
+  Sans,
+  Serif,
+  StackableBorderBox,
+} from "@artsy/palette"
+import { PurchaseHistory_orders } from "__generated__/PurchaseHistory_orders.graphql"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { get } from "Utils/get"
+
+interface OrderRowProps {
+  order: PurchaseHistory_orders["edges"][number]["node"]
+}
+const OrderRow = (props: OrderRowProps) => {
+  const { order } = props
+  const artwork = get(order, o => o.lineItems.edges[0].node.artwork)
+
+  return (
+    artwork && (
+      <Box p={1}>
+        <StackableBorderBox
+          flexDirection="row"
+          alignItems="center"
+          justifyContent="space-between"
+        >
+          <Flex width="350px">
+            <Flex height="auto" alignItems="center">
+              <Image
+                src={get(artwork, a => a.image.resized.url)}
+                width="55px"
+                mr={1}
+              />
+            </Flex>
+            <Flex
+              flexDirection="column"
+              justifyContent="center"
+              style={{ overflow: "hidden" }}
+            >
+              <Link
+                href={`/orders/${order.internalID}/status`}
+                underlineBehavior="hover"
+              >
+                <Serif size="2" weight="semibold">
+                  {artwork.artist_names}
+                </Serif>
+              </Link>
+              <div style={{ lineHeight: "1" }}>
```

Comment:
You don't need this wrapper to adjust line height. You can technically pass that directly to the Serif component...

Though I'm not sure if the types reflect that 🤔

Reply:
👍  Looked at the designs and the line-height is `18px` there so the correct relative line height should be `1.3` but I always thought `<Serif>`'s `size` property automatically sets line height but it wasn't there. Checked order pages and noticed over there the line height is also explicitly set. 🤔

Reply:
Hmmmm... sounds like a bug.